src/customisations/WidgetPermissions.ts

```diff
@@ -36,11 +36,24 @@ async function preapproveCapabilities(
     return new Set(); // no additional capabilities approved
 }
 
+/**
+ * Approves the widget for identity token request, if it can be approved.
```

Comment:
> * Approves the widget for identity token request, if it can be approved.

I'm afraid I don't understand what this sentence means. It seems very circular?

Reply:
yes, it was simplified.